### PR TITLE
Several fixes

### PR DESCRIPTION
**Pull request by sharkykh**
Proposed changes in this pull request:
- e0e4e6b Small typo from my last commit
- 074d9bf Fix for release_group not getting updated in the DB like release_name when media file is manually erased by the user.
- 29155f5 Fix USE_SUBTITLES getting switched from bool to int, and messing up table /history/.
- dfe5d8d Fix for random "AA " lines in log.
Looks like it's the best place to check for this,
because on the more inner log functions I didn't see a better place for this check.
- 5c463f9 Use local Trakt instead of pulling from the site, and a small adjustment.
At least until we start using TheTVDB for posters.

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

**Comment by miigotu** [comment]
Im going to add a few commits to this before merge. You opened a can of worms.

**Comment by sharkykh** [comment]
LOL sorry :stuck_out_tongue_winking_eye:

**Comment by miigotu** [comment]
No its ok, because this is the cause of quite a few hard to diagnose issues.

**Comment by miigotu** [comment]
Tasty? Lol, Please check I didnt fuck up and I'll merge it.

**Comment by miigotu** [comment]
Next is to replace bool\((sickbeard\.[A-Z_]+)\) with $1, but that will take more testing/caution and I'm tired =P

**Comment by miigotu** [comment]
Im ready to merge this if you are ok with that?  Most of my changes were just regex replacements

**Comment by sharkykh** [comment]
I'll start checking now :)

File: sickbeard/__init__.py [hunk 1]
```diff
@@ -865,88 +866,88 @@ def path_leaf(path):
...
+        WEB_ROOT = check_setting_str(CFG, 'General', 'web_root').rstrip("/")
+        WEB_LOG = check_setting_bool(CFG, 'General', 'web_log')
+        WEB_USERNAME = check_setting_str(CFG, 'General', 'web_username', censor_log=True)
+        WEB_PASSWORD = check_setting_str(CFG, 'General', 'web_password', censor_log=True)
         WEB_COOKIE_SECRET = check_setting_str(CFG, 'General', 'web_cookie_secret', helpers.generateCookieSecret(), censor_log=True)
         if not WEB_COOKIE_SECRET:
             WEB_COOKIE_SECRET = helpers.generateCookieSecret()
 
-        WEB_USE_GZIP = bool(check_setting_int(CFG, 'General', 'web_use_gzip', 1))
+        WEB_USE_GZIP = check_setting_bool(CFG, 'General', 'web_use_gzip', True)
```
Comment (sharkykh) [inline sickbeard/__init__.py:878]: This is the first time I see WEB_USE_GZIP.
I just checked, you know the user can't change this from the web UI?
It's not anywhere in gui/slick/*.*
Not that it really matters, just to let you know.

Comment (miigotu) [reply sickbeard/__init__.py:878]: Yes its a config only option for one user whose reverse proxy didnt work with it enabled (he didnt know how to set his proxy up well I guess)

File: sickbeard/__init__.py [hunk 2]
```diff
@@ -1433,64 +1423,56 @@ def path_leaf(path):
...
             if hasattr(curTorrentProvider, 'enable_daily'):
-                curTorrentProvider.enable_daily = bool(check_setting_int(CFG, curTorrentProvider.get_id().upper(),
-                                                                         curTorrentProvider.get_id() + '_enable_daily',
-                                                                         1))
+                curTorrentProvider.enable_daily = check_setting_bool(CFG, curTorrentProvider.get_id().upper(),
+                                                                         curTorrentProvider.get_id() + '_enable_daily', True)
 
             if hasattr(curTorrentProvider, 'enable_backlog'):
-                curTorrentProvider.enable_backlog = bool(check_setting_int(CFG, curTorrentProvider.get_id().upper(),
+                curTorrentProvider.enable_backlog = check_setting_int(CFG, curTorrentProvider.get_id().upper(),
```
Comment (sharkykh) [inline sickbeard/__init__.py]: `check_setting_int` remained on purpose?
(there two more coming up..)

Comment (miigotu) [reply sickbeard/__init__.py]: good catch, nope

File: sickbeard/__init__.py [hunk 3]
```diff
@@ -1433,64 +1423,56 @@ def path_leaf(path):
...
             if hasattr(curTorrentProvider, 'cookies'):
                 curTorrentProvider.cookies = check_setting_str(CFG, curTorrentProvider.get_id().upper(),
-                                                               curTorrentProvider.get_id() + '_cookies', '', censor_log=True)
+                                                               curTorrentProvider.get_id() + '_cookies', censor_log=True)
 
         for curNzbProvider in [curProvider for curProvider in providers.sortedProviderList() if
                                curProvider.provider_type == GenericProvider.NZB]:
-            curNzbProvider.enabled = bool(
-                check_setting_int(CFG, curNzbProvider.get_id().upper(), curNzbProvider.get_id(), 0))
+            curNzbProvider.enabled = check_setting_int(CFG, curNzbProvider.get_id().upper(), curNzbProvider.get_id())
```
Comment (sharkykh) [inline sickbeard/__init__.py]: same as above?

File: sickbeard/__init__.py (hunk 2 above)
Comment (miigotu) [reply sickbeard/__init__.py]: there is another enable_backlog one too that I missed the change on

File: sickbeard/__init__.py (hunk 3 above)
Comment (miigotu) [reply sickbeard/__init__.py]: Yep

File: sickbeard/__init__.py [hunk 4]
```diff
@@ -1433,64 +1423,56 @@ def path_leaf(path):
...
-                curNzbProvider.enable_daily = bool(check_setting_int(CFG, curNzbProvider.get_id().upper(),
-                                                                     curNzbProvider.get_id() + '_enable_daily',
-                                                                     1))
+                curNzbProvider.enable_daily = check_setting_bool(CFG, curNzbProvider.get_id().upper(), curNzbProvider.get_id() + '_enable_daily', True)
 
             if hasattr(curNzbProvider, 'enable_backlog'):
-                curNzbProvider.enable_backlog = bool(check_setting_int(CFG, curNzbProvider.get_id().upper(),
-                                                                       curNzbProvider.get_id() + '_enable_backlog',
-                                                                       curNzbProvider.supports_backlog))
+                curNzbProvider.enable_backlog = check_setting_int(CFG, curNzbProvider.get_id().upper(), curNzbProvider.get_id() + '_enable_backlog',
```
Comment (sharkykh) [inline sickbeard/__init__.py]: and this too?

File: sickbeard/__init__.py (hunk 2 above)
Comment (sharkykh) [reply sickbeard/__init__.py]: haha got that too
I'll fix them

File: sickbeard/__init__.py (hunk 4 above)
Comment (miigotu) [reply sickbeard/__init__.py]: yes, all bool(check_setting_int should now be check_setting_bool

**Comment by miigotu** [comment]
Really cool that you been helping out. It's nice having someone around who doesn't have their head up their ass who can pick out small mistakes like I do =P. Just want to say thanks, it's appreciated.

**Comment by sharkykh** [comment]
Happy to help :smile: 
Hell, I didn't even know how to code in Python when I stumbled upon SickRage a month ago.

**Comment by miigotu** [comment]
I also learned python from working on sickrage haha. Got better at javascript with it also. That was a long time ago now but I just know a bunch of other languages so it was easy to pick up.

File: sickbeard/config.py [hunk 5]
```diff
@@ -322,11 +316,10 @@ def change_version_notify(version_notify):
 
     :param version_notify: New frequency
     """
```
Comment (sharkykh) [inline sickbeard/config.py]: Click Expand and you see it's `change_version_notify`, which doesn't change the frequency but rather
(supposed to) enable or disable the version check thread.

We have that for changing frequency;
```
def change_update_frequency(freq):
    """
    Change frequency of daily updater thread

    :param freq: New frequency
    """
    sickbeard.UPDATE_FREQUENCY = try_int(freq, sickbeard.DEFAULT_UPDATE_FREQUENCY)

    if sickbeard.UPDATE_FREQUENCY < sickbeard.MIN_UPDATE_FREQUENCY:
        sickbeard.UPDATE_FREQUENCY = sickbeard.MIN_UPDATE_FREQUENCY

    sickbeard.versionCheckScheduler.cycleTime = datetime.timedelta(hours=sickbeard.UPDATE_FREQUENCY)
```

So I'll replace it with a code from another thread enable/disable method

**Comment by sharkykh** [comment]
```
2017-02-24 05:55:09 INFO FINDSUBTITLES :: No subtitles to download
2017-02-24 05:55:09 INFO FINDSUBTITLES :: Checking for missed subtitles
2017-02-24 05:55:09 DEBUG FINDSUBTITLES :: Starting new thread: FINDSUBTITLES
AA 
AA 
AA 
AA 
AA 
AA 
2017-02-24 01:55:08 INFO FINDSUBTITLES :: Finished checking for missed subtitles
```

This from the latest develop. When compared to previous logs this is nothing (6 rows in total).
But something's still getting through. At least we know that it comes from FINDSUBTITLES.
I think that the string it's printing isn't empty - maybe just a space.

**Pull request merged**